docs/core_docs/docs/how_to/index.mdx

```diff
@@ -76,6 +76,7 @@ These are the core building blocks you can use when building applications.
 - [How to: stream tool calls](/docs/how_to/tool_streaming)
 - [How to: few shot prompt tool behavior](/docs/how_to/tool_calling#few-shotting-with-tools)
 - [How to: force a specific tool call](/docs/how_to/tool_choice)
+- [How to: init any model in one line](/docs/how_to/chat_models_universal_init/)
```

Comment:
This isn't quite accurate since it doesn't support all models - @baskaryan any other ideas on phrasing or should we just go with it